docs/GCP-development.md

```diff
@@ -24,15 +24,59 @@ for Python and Kind and/or GCP/GKE.
 
 ### GCP Development
 
-Create the GKE Cluster, install Tekton Pipelines, Dashboard and Chains
+Terraform will create the Trillian MySQL database, the Google CA, GKE Cluster and 
+install Tekton Pipelines, Dashboard and Chains via Helm and Sigstore via helm
 
 We are doing a terraform targeted plan/apply because the GKE cluster has to be up and running before helm can apply,
 since we are directly pointing the helm provider to the GKE cluster.
 
+You will need to have gcloud [installed](https://cloud.google.com/sdk/docs/install)
+
 Set the GOOGLE_APPLICATION_CREDENTIALS path to where `gcloud auth application-default login` places your credentials file
 
+Usually that is `$HOME/.config/gcloud/application_default_credentials.json` 
+
+We need to make sure we have the helm charts locally as they are not stored in Artifactory hub.
+
+```bash
+git clone git@github.com:Cray-HPE/sigstore-charts.git $PWD/sigstore-charts
```

Comment:
nit: by default it already puts it in $PWD/sigstore-charts
so not sure it's necessary?